targets.toml

```diff
@@ -409,6 +409,13 @@ cpp = true
 dylib = true
 std = true
 
+[[target]]
+target = "aarch64-unknown-freebsd"
+os = "ubuntu-latest"
+cpp = true
+dylib = true
+std = true
```

Comment:
the ci test failed with

```
     Checking foo v0.1.0 (/home/runner/work/cross/cross/target/tmp/tmp.pcmLAmfkzI)
     Running `rustc --crate-name foo --edition=2021 tmp.pcmLAmfkzI/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat --crate-type lib --emit=dep-info,metadata -C embed-bitcode=no -C debuginfo=2 -C metadata=12b0d6c0320165c1 -C extra-filename=-12b0d6c0320165c1 --out-dir /target/aarch64-unknown-freebsd/debug/deps --target aarch64-unknown-freebsd -C linker=aarch64-unknown-freebsd12-gcc.sh -L dependency=/target/aarch64-unknown-freebsd/debug/deps -L dependency=/target/debug/deps`
error[E0463]: can't find crate for `core`
  |
  = note: the `aarch64-unknown-freebsd` target may not be installed
  = help: consider downloading the target with `rustup target add aarch64-unknown-freebsd`

error[E0463]: can't find crate for `compiler_builtins`

For more information about this error, try `rustc --explain E0463`.
error: could not compile `foo` due to 2 previous errors

Caused by:
  process didn't exit successfully: `rustc --crate-name foo --edition=2021 tmp.pcmLAmfkzI/src/lib.rs --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat --crate-type lib --emit=dep-info,metadata -C embed-bitcode=no -C debuginfo=2 -C metadata=12b0d6c0320165c1 -C extra-filename=-12b0d6c0320165c1 --out-dir /target/aarch64-unknown-freebsd/debug/deps --target aarch64-unknown-freebsd -C linker=aarch64-unknown-freebsd12-gcc.sh -L dependency=/target/aarch64-unknown-freebsd/debug/deps -L dependency=/target/debug/deps` (exit status: 1)
+ rustup component list --toolchain stable-x86_64-unknown-linux-gnu
[cross] warning: rust-std is not available for aarch64-unknown-freebsd
[cross] note: you may need to build components for the target via `-Z build-std=<components>` or in your cross configuration specify `target.aarch64-unknown-freebsd.build-std`
              the available components are core, std, alloc, and proc_macro
Error: Process completed with exit code 101.
```

I do not understand why though, [this](https://github.com/cross-rs/cross/blob/ed713d90087319ff6baf74704a66f21e13625e04/src/rustup.rs#L291-L295) is the check for if the target needs to be installed or not

Reply:
ah, it's tier 3, thought it was t2, can you add `build-std = true` to the target

Reply:
```sh
$ rustup component add rust-std-aarch64-unknown-freebsd --toolchain stable-x86_64-unknown-linux-gnu
error: toolchain 'stable-x86_64-unknown-linux-gnu' does not contain component 'rust-std-aarch64-unknown-freebsd' for target 'x86_64-unknown-linux-gnu'
note: not all platforms have the standard library pre-compiled: https://doc.rust-lang.org/nightly/rustc/platform-support.html
```